mbq/env/__init__.py

```diff
@@ -7,30 +9,73 @@ class EnvException(Exception):
     pass
 
 
+class Environment(Enum):
+    """
+    A simple Enum to provide consistency in how we detect and
+    represent the environment in which a service is running.
+    """
+
+    PRODUCTION = ('prd', 'Production')
+    DEVELOPMENT = ('dev', 'Development')
+    LOCAL = ('lcl', 'Local')
+
+    def __init__(self, short, long):
+        self.short = short
+        self.long = long
```

Comment:
Is `long` a keyword? GitHub's highlighting it

Reply:
It was in Py2